packages/php-db-import-export/tests/functional/Bigquery/ToFinal/IncrementalImportTest.php

```diff
@@ -381,20 +313,13 @@ public function testImportTimestampBehavior(array $features, array $expectedCont
               `name` STRING(50),
               `price` DECIMAL,
               `isDeleted` INT64,
-              `_timestamp` TIMESTAMP
+              `_timestamp` TIMESTAMP,
+               PRIMARY KEY(id) NOT ENFORCED
```

Comment:
ten PK se z nejakeho duvodu nenastavoval, ale pred tim to asi nevadilo, bo porovnavani hodnot bylo rizene ficurou. ted uz neni a je ten PK tam realne potreba

Reply:
S feature to nemá nic společného o řádek níž sa přidával ten PK a tys to smazal :D

Reply:
smazal, protoze to realne ten PK nepridavalo. nevim proc, ale proste do tableDefinition to doslo bez PK

Reply:
a proto jsem to pridal sem. S ficurou to melo podle me spolecne to, ze ta ficura ovlivnovala to, zda se PK realne pouzilo nebo ne. Ale na to se musim jeste podivat hloubeji

Reply:
Tak to nevím teď sem to testoval u sebe a když smažu ten PRIMARY KEY a přidám tam ten update co tam byl tak ten test mě prošel, nemáš třeba staré závislosti?

Reply:
nevylucuju, zkusim

Reply:
hele tak to asi bylo zavislostma. `Upgrading google/cloud-bigquery (v1.24.0 => v1.31.0)` a uz to frci, toto teda vratim